metadata-protobuf/proto/Video.proto

```diff
@@ -85,4 +87,11 @@ message VideoMetadata {
 message VideoCategoryMetadata {
     // Category name
     optional string name = 1;
+}
+
+message ContentMetadata {
```

Comment:
I think this is a good idea in general, but it's not backwards compatible. All the videos that are already on chain use regular `VideoMetadata` for their metadata. I guess we can keep this new `ContentMetadata` that will be used for new stuff but QN mappings will still need to handle regular `VideoMetadata`.